staging/src/k8s.io/apiserver/pkg/server/dynamiccertificates/dynamic_sni_content.go

```diff
@@ -36,7 +40,8 @@ func NewDynamicSNIContentFromFiles(purpose, certFile, keyFile string, sniNames .
 		DynamicCertKeyPairContent: servingContent,
 		sniNames:                  sniNames,
 	}
-	if err := ret.loadCertKeyPair(); err != nil {
+	ctx := context.TODO()
```

Comment:
Same comment as above: add `ctx`.

Reply:
same as above

have to leave one context.background for NewDynamicSNIContentFromFiles in

staging/src/k8s.io/apiserver/pkg/server/options/serving.go